src/components/Draggable/util.tsx

```diff
@@ -0,0 +1,35 @@
+import { ActualQuantity } from './types/ActualQuantity'
+
+/**
+ * Checks which direction the arrow key was pressed
+ *
+ * @param key The key pressed
+ * @returns The direction (up/down/right/left)
```

Comment:
I don't know if is a good practice but this will returns `undefined` too if not match anything
```suggestion
 * @returns The direction (up/down/right/left) or undefined
```